src/main/kotlin/org/rust/ide/refactoring/inlineFunction/RsInlineFunctionDialog.kt

```diff
@@ -73,4 +73,6 @@ class RsInlineFunctionDialog(
 
     override fun getHelpId(): String =
         "refactoring.inlineMethod"
+
+    override fun isInlineThis(): Boolean = false
```

Comment:
Does this change something in the UI or behaviour? Maybe we could add a comment here

Reply:
I removed this method, it was accidentally overwritten